core/app/modules/kit-library/data/taxonomies/controller.php

```diff
@@ -2,45 +2,37 @@
 namespace Elementor\Core\App\Modules\KitLibrary\Data\Taxonomies;
 
 use Elementor\Core\App\Modules\KitLibrary\Data\Base_Controller;
-use Elementor\Core\App\Modules\KitLibrary\Data\Exceptions\Wp_Error_Exception;
 
 if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Exit if accessed directly.
 }
 
 class Controller extends Base_Controller {
+
 	public function get_name() {
 		return 'kit-taxonomies';
 	}
 
-	public function register_endpoints() {
-		//
-	}
-
-	/**
-	 * Register internal endpoint.
-	 */
-	protected function register_internal_endpoints() {
-		// Register as internal to remove the default endpoint generated by the base controller.
-		$this->register_endpoint( Endpoints\Index::class );
+	public function get_items_args( $methods ) {
+		return [
+			'force' => [
+				'description' => 'Force an API request and skip the cache.',
+				'required' => false,
+				'default' => false,
+				'type' => 'boolean',
+			],
+		];
 	}
 
-	/**
-	 * @param \WP_REST_Request $request
-	 *
-	 * @return \WP_Error|array
-	 */
 	public function get_items( $request ) {
-		try {
-			$data = $this->get_repository()->get_taxonomies( $request->get_param( 'force' ) );
-		} catch ( Wp_Error_Exception $exception ) {
-			return new \WP_Error( $exception->getCode(), $exception->getMessage(), [ 'status' => $exception->getCode() ] );
-		} catch ( \Exception $exception ) {
-			return new \WP_Error( 'server_error', __( 'Something went wrong.', 'elementor' ), [ 'status' => 500 ] );
-		}
+		$data = $this->get_repository()->get_taxonomies( $request->get_param( 'force' ) );
 
 		return [
 			'data' => $data->values(),
 		];
 	}
+
+	public function register_endpoints() {
```

Comment:
Make it default (not abstract)

Reply:
Handled: https://github.com/elementor/elementor/pull/14205/commits/86a144e3ede79b7e613a4aafcfc4b0a22b7dace8